test/JsonParser.test.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
```

Comment:
This is a unittest -> rename it to `test/JsonParser.unittest.js`.

You still need to add a integration test.

Best add it to this test case: https://github.com/webpack/webpack/tree/master/test/cases/parsing/bom